### DISCUSSION TITLE
Increase Native Tokens total supply.

### DISCUSSION THREAD

**Discussion by KhalilAsghar544** [cardano-foundation/developer-portal#1029] [Q&A] [answered]
Hello,
I want to know that can we increase the native tokens' initial supply, or can we mint more tokens with old keys that were initially used for minting a number of tokens earlier?

**Comment by OlofBlomqvist** [comment, marked as answer]
Hi! This is not the right place for technical questions, but the answer is that **it depends on your minting policy**.
As a starting point, there is some basic documentation around minting policies here: https://cardano-ledger.readthedocs.io/en/latest/explanations/policies.html.

For further questions I recommend you instead use the forum or discord servers here:

- [Cardano Forum](https://forum.cardano.org)
- [Community Discord](https://discord.gg/a4JHg9AG)
- [IOG Technical Discord](https://discord.gg/inputoutput)

**Reply by rphair** [reply]
and @OlofBlomqvist @KhalilAsghar544 don't forget... 🤓
- [Cardano StackExchange](https://cardano.stackexchange.com/)
